app/containers/LanguageProvider/index.js

```diff
@@ -24,11 +24,7 @@ const stateSelector = createSelector(
 export default function LanguageProvider(props) {
   const { locale } = useSelector(stateSelector);
   return (
-    <IntlProvider
-      locale={locale}
-      key={locale}
-      messages={props.messages[locale]}
-    >
+    <IntlProvider locale={locale} messages={props.messages[locale]}>
```

Comment:
I believe `key` is here because of https://github.com/formatjs/react-intl/issues/234#issuecomment-163366518 , I've run into the same issue before.  If `react-intl` v3 uses the new context API, this can probably be removed.

Reply:
Switching locales works just fine without the key even though react-intl v2.9 does use the old Context API.

This issue is from 2015 though so it's probably been resolved since then, unintentionally or not, via the many changes that occurred in react, react-redux and/or react-intl.

And in any case we're switching to v3 which uses the new Context API.

If that makes sense, please approve so I can merge?

Reply:
kk 👍